pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -162,21 +152,9 @@ def _configure_model_summary_callback(
             )
             return
 
-        if weights_summary == "top":
-            # special case the default value for weights_summary to preserve backward compatibility
-            max_depth = 1
-        else:
-            rank_zero_deprecation(
-                f"Setting `Trainer(weights_summary={weights_summary})` is deprecated in v1.5 and will be removed"
-                " in v1.7. Please pass `pytorch_lightning.callbacks.model_summary.ModelSummary` with"
-                " `max_depth` directly to the Trainer's `callbacks` argument instead."
-            )
-            if weights_summary not in ModelSummaryMode.supported_types():
-                raise MisconfigurationException(
-                    f"`weights_summary` can be None, {', '.join(ModelSummaryMode.supported_types())}",
-                    f" but got {weights_summary}",
-                )
-            max_depth = ModelSummaryMode.get_max_depth(weights_summary)
+        # If the user wants to configure a model summary callback, without explicitly passing it,
+        # to the callbacks list, we will create it with max depth of 1
+        max_depth = 1
```

Comment:
@rohitgr7 @akihironitta I have set the default value of max_depth as 1 if one does not pass a model summary callback but uses `enable_model_summary.` Let me know if this makes sense.

Reply:
Yes, that's correct because the previous default "top" `means max_depth=1`. But the model summary already has the `max_depth=1` as default, so this code can be removed (I would also remove the comment).

Reply:
@awaelchli Changed it, can you have a look?